SUPPORT_POLICY.md

```diff
@@ -0,0 +1,29 @@
+Distroless is currently tracks debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
```

Comment:
```suggestion
Distroless currently tracks debian on their [standard support timeline](https://wiki.debian.org/DebianReleases#Production_Releases).
```

Reply:
good catch, I'm going to edit these in directly, sometimes the CLA bot gets confused with multiple authors